introduction.md

```diff
@@ -1,3 +1,6 @@
 Zeljko Markovic
 
-JavaScript, HTML & CSS
\ No newline at end of file
+JavaScript, HTML & CSS
+
+* javascript
+* html
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?